cluster/manifests/e2e-resources/pool-reserve.yaml

```diff
@@ -1,5 +1,5 @@
 {{ if eq .Cluster.Environment "e2e" }}
-{{ range $pool := split "default-worker-splitaz,worker-limit-az,worker-combined,worker-instance-storage,worker-node-tests,worker-karpenter,worker-arm64" "," }}
+{{ range $pool := split "worker-limit-az,worker-instance-storage,worker-node-tests,worker-arm64" "," }}
```

Comment:
I think we need to shorten this list as well: https://github.com/zalando-incubator/kubernetes-on-aws/pull/9120/files#diff-b5504898e28d320aa7ff060453dc48970aa8c4995a1350651150bc65cdabf4e0R39-R46

Because the Deployments that are checked there won't exist.

Reply:
Fixed